platform/mime/core/platform-mime-core-impl/src/main/java/ddf/mime/mapper/MimeTypeMapperImpl.java

```diff
@@ -66,8 +71,7 @@ public int compare(MimeTypeResolver o1, MimeTypeResolver o2) {
     /**
      * Constructs the MimeTypeMapper with a list of {@link MimeTypeResolver}s.
      *
-     * @param mimeTypeResolvers
-     *            the {@link List} of {@link MimeTypeResolver}s
+     * @param mimeTypeResolvers the {@link List} of {@link MimeTypeResolver}s
```

Comment:
There is a TODO comment left in some code in another place in the file. It's not in anything you touched, but it would be a bonus if we could clean it up:

` // TODO: This is to force the TikaMimeTypeResolver to be called`